client/src/index.js

```diff
@@ -11,6 +11,7 @@ const root = ReactDOM.createRoot(document.getElementById("root"));
 const store = configureStore({
   reducer: {
     auth: reducer.auth,
+    request: reducer.request,
```

Comment:
Can you pull new code and update with new code here, might appear unappropriated code if merge

Reply:
Done